.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,122 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
you should add a .gitignore that does not include .idea directories and other things that might not be needed